### PR TITLE
Feat/ldp migration

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | yes
| Tickets          | MANAGER-4011
| License          | BSD 3-Clause

## Description

Rework of https://github.com/ovh/manager/pull/2744 due to conflicts

## Related
- [x] TRANS-28872

### CODE REVIEW

File: packages/manager/modules/dbaas-logs/src/logs/detail/home/home.module.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import angular from 'angular';
+
+import '@ovh-ux/manager-core';
+import '@ovh-ux/ng-ovh-cloud-universe-components';
+import '@ovh-ux/ng-ovh-sidebar-menu';
```
Comment (Heiiji) [inline packages/manager/modules/dbaas-logs/src/logs/detail/home/home.module.js:5]: No peerdep required for `@ovh-ux/ng-ovh-sidebar-menu` ? :)

Comment (marie-j) [reply packages/manager/modules/dbaas-logs/src/logs/detail/home/home.module.js:5]: modules shouldn't need the sidebar menu, I'll remove the import

**Review by Heiiji — DISMISSED** [review]

File: packages/manager/apps/dedicated/client/app/app.module.js [hunk 2]
```diff
@@ -56,6 +56,7 @@ import 'ovh-angular-responsive-tabs';
 import 'ckeditor';
 import 'script-loader!messenger/build/js/messenger.min.js';
 import 'script-loader!filesize/lib/filesize.js';
+import 'script-loader!angular-websocket/dist/angular-websocket';
```
Comment (JeremyDec) [inline packages/manager/apps/dedicated/client/app/app.module.js:59]: No `// eslint-disable-line` required ?

File: packages/manager/apps/dbaas-logs/webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+const _ = require('lodash');
+const path = require('path');
+const webpack = require('webpack'); // eslint-disable-line
```
[the remark this answers is not in the document]
Comment (marie-j) [reply packages/manager/apps/dbaas-logs/webpack.config.js:3]: This was auto-generated with `generate:app` command, I'll have a look

File: packages/manager/apps/dedicated/client/app/app.module.js (hunk 2 above)
Comment (marie-j) [reply packages/manager/apps/dedicated/client/app/app.module.js:59]: As there are multiple script loader imports, it was disabled for the batch: https://github.com/ovh/manager/blob/3aea938e5b5a4528d80111b29fcbe73271b3ed9b/packages/manager/apps/dedicated/client/app/app.module.js#L1

**Review by JeremyDec — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report test#15975.0 ✘  
* Stage 1
  * lint ✔ 
  * SemVer ✘

**Comment by ovh-cds** [comment]
CDS Report test#15978.0 ✘  
* Stage 1
  * lint ✔ 
  * SemVer ✘

**Comment by ovh-cds** [comment]
CDS Report build#16218.0 ✘  
* 
  * Build ✘

**Comment by marie-j** [comment]
I've rewritten history to release Dbass-logs app as a 1.0.0

**Review by antleblanc — DISMISSED** [review]

File: packages/manager/apps/dbaas-logs/package.json [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    "angular-i18n": "^1.7.8",
+    "angular-resource": "^1.7.8",
+    "angular-sanitize": "^1.7.8",
+    "angular-translate": "^2.18.1",
+    "angular-translate-loader-pluggable": "^1.3.1",
+    "angular-ui-bootstrap": "~1.3.3",
+    "angular-websocket": "^2.0.1",
+    "bootstrap": "~3.3.7",
+    "chart.js": "^2.8.0",
+    "flatpickr": "~4.5.2",
```
Comment (antleblanc) [inline packages/manager/apps/dbaas-logs/package.json:58]: Relates to #3881.

```suggestion
    "flatpickr": "^4.6.3",
```

**Review by antleblanc — CHANGES_REQUESTED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**